src/credentials/CHIPCertFromX509.cpp

```diff
@@ -693,5 +694,40 @@ DLL_EXPORT CHIP_ERROR ConvertX509CertToChipCert(const uint8_t * x509Cert, uint32
     return err;
 }
 
+CHIP_ERROR ConvertX509CertsToChipCertArray(const ByteSpan & x509Cert1, const ByteSpan & x509Cert2, uint8_t * chipCertArrayBuf,
```

Comment:
Where opcerts are used, like in AddOpCert, the langue for order is:

```
[[ref_OpCertStructOperationalCert]]
====== OperationalCert
This field SHALL contain the associated <<ref_NOC, Node Operational Credential>> chain for the particular fabric referred by `FabricIndex`, encoded as a <<ref_oce-chip-cert-array, chip-certificate-array>>.
The array SHALL contain components of the chain in the following order:

- <<ref_NOC,Node Operational Certificate (NOC)>>
- <<ref_ICAC,Intermediate CA (ICA) Certificate>>, if present
```

So it's from leaf to root.

@emargolis Should we update the following spec section to clarify order:

```
[[ref_oce-chip-cert-array]]
==== CHIP Certificate Array
```
?

Reply:
https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/3345 has order clarifications in it.